rl_zoo3/train.py

```diff
@@ -152,6 +153,7 @@ def train():
         default=False,
         help="if toggled, display a progress bar using tqdm and rich",
     )
+    parser.add_argument("--tags", type=str, default="", nargs="+", help="Tags for wandb run, e.g.: --tags optimized pr-123")
```

Comment:
```suggestion
    parser.add_argument("--tags", type=str, default=[], nargs="+", help="Tags for wandb run, e.g.: --tags optimized pr-123")
```

I would also think renaming it `-tags` and `--wandb-tags` for consistency.